phost/src/main.rs

```diff
@@ -348,6 +348,20 @@ async fn batch_sync_block(
     Ok(synced_blocks)
 }
 
+async fn get_latest_sequence(client: &XtClient) -> Result<u32, Error> {
+    let block_tip = get_block_at(&client, None, false).await?.block;
+    let hash = client.block_hash(Some(subxt::BlockNumber::from(block_tip.block.header.number))).await?;
+    let key = storage_value_key_vec("PhalaModule", "Sequence");
```

Comment:
Why not use the high level `client.fetch_or_default` api? Please either do it or add: `TODO: switch to high level fetch_or_default api`